src/doc/reference.md

```diff
@@ -2079,6 +2079,8 @@ The following configurations must be defined by the implementation:
 * `target_family = "..."` - Operating system family of the target, e. g.
   `"unix"` or `"windows"`. The value of this configuration option is defined
   as a configuration itself, like `unix` or `windows`.
+* `target_has_atomic = "..."` - Set of integer sizes on which the target can perform
+  atomic operations. Values are `"8"`, `"16"`, `"32"`, `"64"` and `"ptr`".
```

Comment:
Is `ptr` the word we want to use here? AFAIK, it isn't used anywhere else in the language (although there are examples in `std`) when pointers are discussed e.g. `target_pointer_width`, `isize`/`usize`, `*const`/`*mut`.